File: src/main/java/com/uwetrottmann/trakt5/entities/BaseMovie.java
package com.uwetrottmann.trakt5.entities;

import org.threeten.bp.OffsetDateTime;

public class BaseMovie {

    public Movie movie;

    public OffsetDateTime collected_at;
    public OffsetDateTime last_watched_at;
    public OffsetDateTime listed_at;
    public int plays;

}

File: src/main/kotlin/com/redissi/trakt/entities/BaseMovie.kt
package com.redissi.trakt.entities

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import java.time.OffsetDateTime

@JsonClass(generateAdapter = true)
class BaseMovie(
    val movie: Movie? = null,
    @Json(name = "collected_at")
    val collectedAt: OffsetDateTime? = null,
    @Json(name = "last_watched_at")
    val lastWatchedAt: OffsetDateTime? = null,
    @Json(name = "last_updated_at")
    val lastUpdatedAt: OffsetDateTime? = null,
    @Json(name = "listed_at")
    val listedAt: OffsetDateTime? = null,
    val plays: Int = 0
)